set_user.c

```diff
@@ -318,12 +318,15 @@ set_user(PG_FUNCTION_ARGS)
 				/* can only escalate with set_user_u */
 				ereport(ERROR,
 						(errcode(ERRCODE_INSUFFICIENT_PRIVILEGE),
-						 errmsg("Switching to superuser only allowed for privileged procedure: \'set_user_u\'")));
+						 errmsg("Switching to superuser only allowed "
```

Comment:
Per the error message guidelines, this message should be more like "switching to superuser not allowed" with perhaps an errhint suggesting "use set_user_u() instead". Note that errmsg should not start with a capitalized word, nor be a complete sentence, also per the guidelines. See https://www.postgresql.org/docs/current/static/error-style-guide.html